### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by kiendang** [julie-is-late/aur-libnvidia-container#10]
Hi, not an actual issue, just wanna drop a note. Recently the machine I run arch on has been inactive due to some technical issues so I've not been able to keep the packages up to date. I hope to resolve these issues within a month or so. Thanks a lot for taking care of things for a while now!

**Comment by julie-is-late** [comment]
Hey, no worries. Been having some personal stuff go on in my life too so I wasn't exactly fast at fixing the package haha. 
I'll go ahead and bump the bin packages too now that you have me thinking about it, though.

**Comment by kiendang** [comment]
> I'll go ahead and bump the bin packages too now that you have me thinking about it

Thanks for this. Now I feel bad for unintentionally rushing you lol.

> Been having some personal stuff go on in my life

Hope everything's going well for you!

**Comment by kiendang** [comment]
Hey how it's going?

Sorry again for the long delay. What I thought was a simple issue turned out to be either my CPU/mb being broken. And due to them being so outdated (was using Sandy Bridge lol) I basically had to replace the whole PC. So ya, couldn't settle that in a month like I said back when I opened this issue.

I'm building a new PC tho. Has ordered all the parts. Should have a working system within uhmm... a month or so. Maybe.

**Comment by julie-is-late** [comment]
haha you're good, i've gone through similar hardware journeys in the past and it always is far more work than I expect it to be :pensive:

The packages have been fine so far, though there was some work needed a few months ago because of some c-group shenanigans that had me [picking a fight with a random person on the internet](https://aur.archlinux.org/packages/nvidia-container-toolkit#comment-821275) haha. It's all resolved now though :)

There is also some weirdness with the binary package for `-runtime`. basically the runtime package is just a stub now and everything it used to do is now done in the `-toolkit`, so I think they haven't cut a binary release of it? anyways the RPM for it hasn't been updated in a long time, and the website makes it annoyingly hard to figure out what build versions are actually pushed. I'm thinking of removing the binary package for it or trying to figure out if I can see internally what's going on with the versioning, but I just haven't had time to look into it. 

Things are going well though! glad to hear you're getting your pc sorted :)

**Comment by kiendang** [comment]
Ya I did see that exchange. I guess that comes with maintaining anything open source. You handled that really well tho.

I do vaguely remember reading something about deprecating `-runtime` when `-toolkit` was first released. About the binary thing, earlier when I was maintaining them every time I had to update them it was kind of tedious and I don't think many people if not any use them instead of the source code version anyway. Definitely won't miss them if we decide to just remove them.

> Things are going well though! glad to hear you're getting your pc sorted :)

Glad to hear :). My parts have started to arrive. Won't have to wait much longer.

**Comment by julie-is-late** [comment]
yeah definitely a situation of just can't make everyone happy, lol. like if you have an issue with this, don't shoot the messenger lol. 

and yay for parts!

**Comment by kiendang** [comment]
So I got my new machine up and running. Just made a bunch of PRs for the new releases. Could help me double check? Thanks!

**Comment by kiendang** [comment]
I just pushed the new versions to aur.
